client/commonFramework/report-issue.js

```diff
@@ -5,7 +5,7 @@ window.common = (function({ common = { init: [] } }) {
         'Challenge [',
         (common.challengeName || window.location.pathname),
         '](',
-        window.location.href,
```

Comment:
This file is no longer in use. 

Please edit the [new version of this file](https://github.com/FreeCodeCamp/FreeCodeCamp/blob/549cfc305f75d8c6036a3cde65c47b518607b18c/common/app/routes/challenges/redux/bug-saga.js). Line 53 is where the location is being sent with the issue :+1: